provider/provider.go

```diff
@@ -1350,22 +1368,19 @@ func setFieldViaOp(typ string, attrs map[string]interface{}, name string, key st
 	var diags diag.Diagnostics
 
 	valStr := attrValueString(typ, key, val, attrs)
+
 	op := fmt.Sprintf("%s.%s = %s", name, key, valStr)
 
-	if typ == "dashboard" {
```

Comment:
Is it intended that this code is removed?

Reply:
Good catch, it is not.